modules/calib3d/include/opencv2/calib3d.hpp

```diff
@@ -2192,7 +2362,7 @@ CV_EXPORTS Mat findFundamentalMat( InputArray points1, InputArray points2,
 @param points1 Array of N (N \>= 5) 2D points from the first image. The point coordinates should
 be floating-point (single or double precision).
 @param points2 Array of the second image points of the same size and format as points1 .
-@param cameraMatrix Camera matrix \f$K = \vecthreethree{f_x}{0}{c_x}{0}{f_y}{c_y}{0}{0}{1}\f$ .
+@param cameraMatrix Camera matrix \f$A = \vecthreethree{f_x}{0}{c_x}{0}{f_y}{c_y}{0}{0}{1}\f$ .
```

Comment:
Do we really need to rename this matrix?

It is already used in samples, like [here](https://github.com/opencv/opencv/blob/4.2.0/samples/cpp/stitching_detailed.cpp#L562). Also there are [wiki](https://en.wikipedia.org/wiki/Camera_resectioning), external [articles](http://ksimek.github.io/2013/08/13/intrinsic/).

I believe there is higher chance to find documentation about "K" camera matrix rather than about "A" (due false positive cases).

Reply:
Yes, I fully agree. Maybe one day we should take the courage and change the name in the section's introduction and accordingly in its functions. maybe :)